### ISSUE TITLE
Training model

### ISSUE THREAD

**Issue by ZhihongChen123** [Mhaiyang/VisDrone#4]
I tried to train the model, but the rpn_bbox_loss remained at around 700 and the model could not converge.Please help me, thank you.

**Comment by Haotian-Zhang** [comment]
Hi, I had the same issue above. Could you please help us with it?

**Comment by katyamineeva** [comment]
@ZhihongChen123 @Haotian-Zhang did you succeeded in training the model? I run train_visdrone.py. After both 1, 5, and 20 epochs the model is no longer able to detect anything. I can only guess, what is the reason:

1. There is a bug in the code.
2. All weights of the model have to be fine-tuned (commented block of code)
3. Due to the constant learning rate, the optimization doesn't coverage. I'll try something like ReduceLROnPlateau in my further attempts.

@Mhaiyang please, share some more details of the training process?
